inc/console/cache/configurecommand.class.php

```diff
@@ -0,0 +1,216 @@
+ * Copyright (C) 2003-2014 by the INDEPNET Development Team.
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * GLPI is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * GLPI is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Console\Cache;
+
+if (!defined('GLPI_ROOT')) {
+   die("Sorry. You can't access this file directly");
+}
+
+use Glpi\Cache\CacheManager;
+use Glpi\Console\AbstractCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @since 10.0.0
+ */
+class ConfigureCommand extends AbstractCommand {
+
+   /**
+    * Error code returned if cache configuration file cannot be write.
+    *
+    * @var int
+    */
+   const ERROR_UNABLE_TO_WRITE_CONFIG = 1;
+
+   protected $requires_db = false;
+
+   /**
+    * Cache manager.
+    * @var CacheManager
+    */
+   private $cache_manager;
+
+   public function __construct(string $name = null) {
+      $this->cache_manager = new CacheManager();
+
+      parent::__construct();
+   }
+
+   protected function configure() {
+
+      $this->setName('glpi:cache:configure');
+      $this->setAliases(['cache:configure']);
+      $this->setDescription('Define cache configuration');
+
+      $this->addOption(
+         'context',
+         null,
+         InputOption::VALUE_REQUIRED,
+         __('Cache context (i.e. \'core\' or \'plugin:plugin_name\')'),
+         'core'
+      );
+
+      $this->addOption(
+         'dsn',
+         null,
+         InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY,
+         __('Cache system DSN')
+      );
+
+      $this->addOption(
+         'namespace',
+         null,
+         InputOption::VALUE_REQUIRED,
+         __('Cache namespace')
+      );
+
+      $this->addOption(
+         'use-default',
+         null,
+         InputOption::VALUE_NONE,
+         __('Unset cache configuration to use default filesystem cache for given context')
+      );
+
+      $this->addOption(
+         'skip-connection-checks',
+         null,
+         InputOption::VALUE_NONE,
+         __('Skip connection checks')
+      );
+
+      $this->addUsage('--use-default');
+      $this->addUsage('--dsn=file://path/to/cache');
```

Comment:
@trasher thinks that this is not really usefull, as filesystem cache location is already updatable using the `GLPI_CACHE_DIR` constant, and having another way to configure it may lead to pebkac issues.

So we may remove ability to configure a specific `file://` DSN and indicates user to run `bin/console cache:configure --use-default [--context=core]` to use the default filesystem cache (this is already implemented).